submissions/xT0PERx/Popup-style/style.css

```diff
@@ -4,42 +4,126 @@
     flex-direction: row;
     justify-content: flex-end;
     font-size: 1em;
-    text-align: center;
     padding: 5px;
 }
 
+.container ol {
+    display: flex;
+    flex-flow: wrap row;
+}
+.container li {
+    display: inline-block;
+    width: 50px;
+    height: 50px;
+    margin: 5px;
+    text-align: center;
+}
+
 .container div {
     width: 80px;
     height: 80px;
     margin: 5px;
     cursor: pointer;
 }
 
+.main_tullbar {
+    position: relative;
+}
+
 img {
-    width: 40px;
-    height: 40px;
+    width: 32px;
+    height: 32px;
 }
 
-label img:hover {
-    border: 1px solid #1d92bd;
-    border-radius: 20%;
+.menu {
+    position: absolute;
+    background-image: url(./images/Menu.png);
+    width: 32px;
+    height: 32px;
+    cursor: pointer;
+    right: 4px;
+    top: 3px;
+}
+
+.check_input {
+    -webkit-appearance: none;
+}
+
+/*                                                      Normal */
+
+.mail p,
+.images p,
+.menu,
+.menu_more,       
+.home img,
+.avatar img {
+    text-decoration: none;
+    border: 1.5px solid transparent;
+}
+
+.menu {
+    border: 1.5px solid white
+}
+
+.mail,
```

Comment:
If you don't add some specific styles for each class no need to create unique class names for similar elements. Just create one class name for all menu items. and use him.